its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -956,7 +956,7 @@ public void incrementalPrAnalysis_ProducesUnchangedFiles() throws IOException {
     Assume.assumeTrue(ORCHESTRATOR.getServer().version().isGreaterThanOrEquals(9, 4)); // Cache API was introduced in 9.4
 
     String projectKey = "incremental-pr-analysis";
-    String baseBranch = "main";
+    String baseBranch = TestUtils.getMainBranchName(ORCHESTRATOR);
```

Comment:
What about this? Bit more wordy, also more clear. As it might look like we want main branch of some project or analysis
```suggestion
    String baseBranch = TestUtils.getDefaultMainBranchName(ORCHESTRATOR);
```
or maybe like this
```suggestion
    String baseBranch = TestUtils.getDefaultBranchName(ORCHESTRATOR);
```